Tabloid/client/src/components/TagList.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from "react";
+import Tag from "./Tag";
+import { getAllTags } from "../modules/tagManager";
+
+export default function TagList() {
+  const [ tags, setTags ] = useState([]);
+
+  useEffect(() => {
+    getAllTags().then(setTags);
```

Comment:
Consider creating a function that gets and sets tags, and calling that function in this useEffect instead. That function can be used again when you need to update state after editing or deleting a tag later on.

Reply:
good idea!

Reply:
Changes made.